projects/canopy/src/lib/breadcrumb/breadcrumb-item/breadcrumb-item.component.scss

```diff
@@ -16,6 +16,16 @@
     }
   }
 
+  &__container--hide-icons {
+    .lg-breadcrumb-item__content .lg-icon {
+      display: none;
+
+      @include lg-breakpoint('md') {
+        display: inline;
```

Comment:
I've forgotten this one, icons are visible on medium always? In hindsight maybe we should have called the modifier `--compact` or similar.

Reply:
not sure i follow, sorry! are you able to elaborate a tad?

Reply:
sorry yes I didn't explain that very well, I think I was slightly at odds with a class called..
`&__container--hide-icons`
that contained a block where it shows the icons...
```
@include lg-breakpoint('md') {
        display: inline;
}
```
I think partly it might be a slight lack of understanding on my side.